pkg/apiserver/api_installer.go

```diff
@@ -104,9 +104,9 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	admit := a.group.Admit
 	context := a.group.Context
 
-	serverGroupVersion := a.group.GroupVersion
+	externalGroupVersion := a.group.GroupVersion
 	if a.group.ServerGroupVersion != nil {
-		serverGroupVersion = *a.group.ServerGroupVersion
+		externalGroupVersion = *a.group.ServerGroupVersion
```

Comment:
> In usage below, you use it as the externalVersion that will be decoded to match the internalVersion of the object you're interpretting. Otherwise, you're decoding to serverVersion and converting that to internalVersion.

Hmm..I'm not sure I follow your argument here. Are you trying to say if we decode data to a version, then that version should be called `externalVersion`?